AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -52,37 +52,36 @@
 /// the property cannot be immediately applied and the node does not already have pending changes.
```

Comment:
Assert the lock is held?  This can get nasty if there is a race with the __loaded optimization